sda/conode_test.go

```diff
@@ -0,0 +1,30 @@
+package sda
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConode_ProtocolRegisterName(t *testing.T) {
```

Comment:
If you could provide tests for insantiating that'd be nice. It's two small functions so we could aim for high coverage for them.